02-oop-inheritance/javaScript/downloableMovie.js

```diff
@@ -0,0 +1,59 @@
+function MovieObserver() {
+
+  this.playing = function (title) {
```

Comment:
All functions assigned to a "Class" should be defined into the prototype not into the constructor